governance/default.go

```diff
@@ -543,8 +560,6 @@ func (g *Governance) ParseVoteValue(gVote *GovernanceVote) (*GovernanceVote, err
 		} else {
 			val = false
 		}
-	default:
-		logger.Warn("Unknown key was given", "key", k)
```

Comment:
Why this line is removed? I think leaving log is better.

Reply:
This log is also logged at the front of the `ParseVoteValue` function, so I removed it.
```
func (g *Governance) ParseVoteValue(gVote *GovernanceVote) (*GovernanceVote, error) {
	var val interface{}
	k, ok := GovernanceKeyMap[gVote.Key]
	if !ok {
		logger.Warn("Unknown key was given", "key", k)
		return nil, ErrUnknownKey
	}
```